pkg/master/master.go

```diff
@@ -202,6 +213,10 @@ type Master struct {
 	// "Outputs"
 	Handler         http.Handler
 	InsecureHandler http.Handler
+
+	// Used for secure proxy
+	tunnels       util.SSHTunnelList
+	installSSHKey InstallSSHKey
```

Comment:
Need to pass in the config's InstallSSHKey into the master struct's installSSHKey in New() below.